app/models/manageiq/providers/openstack/network_manager/cloud_subnet.rb

```diff
@@ -55,13 +55,12 @@ def with_provider_object
     super(connection_options)
   end
 
-  private
-
   def self.connection_options(cloud_tenant = nil)
     connection_options = {:service => "Network"}
     connection_options[:tenant_name] = cloud_tenant.name if cloud_tenant
     connection_options
   end
+  private_class_method :connection_options
```

Comment:
I could be wrong though.  Either way, I think connection_options was changed over there in a different file and might be public.  Please see the other PR and let me know what you think.